src/SFA.DAS.FindEpao.UnitTests/Application/Courses/Queries/GetCourseEpaos/WhenHandlingGetCourseEpaosQuery.cs

```diff
@@ -37,12 +40,23 @@ public class WhenHandlingGetCourseEpaosQuery
                 .Setup(client => client.Get<GetStandardsListItem>(
                     It.Is<GetStandardRequest>(request => request.StandardId == query.CourseId)))
                 .ReturnsAsync(coursesApiResponse);
+            foreach (var epao in epaoApiResponse)
+                mockAssessorsGetStandardsApiClient
+                    .Setup(client => client.Get<IEnumerable<GetStandardsExtendedListItem>>(
+                        It.Is<GetCourseEpaosStandardVersionsRequest>(request => request.LarsCode == query.CourseId && 
+                        request.organisationId == epao.EpaoId)))
+                    .ReturnsAsync(standardsResponse);
             mockCourseEpaoFilter
                 .Setup(service => service.IsValidCourseEpao(It.IsAny<GetCourseEpaoListItem>()))
                 .Returns(true);
+            mockCourseEpaoFilter
+                .Setup(service => service.ValidateVersionDates(It.IsAny<DateTime>(), It.IsAny<DateTime>()))
+                .Returns(true);
+
 
             var result = await handler.Handle(query, CancellationToken.None);
 
+            result.Course.StandardVersions.Should().HaveCountGreaterOrEqualTo(1);
```

Comment:
Should this not be equivalent to what is in `standardsResonse` as that's what populates the versions of all the courses?